app/src/main/res/resource/values-de/strings.xml

```diff
@@ -56,20 +56,22 @@
     <string name="unsplash">Unsplash.com</string>
     <string name="about_unsplash">Freie (mach, was immer du willst) hochauflösende Fotos.</string>
     <string name="retrofit">Retrofit 2.0</string>
-    <string name="about_retrofit">Type-safe HTTP client für Android und Java von Square, Inc.</string>
+    <string name="about_retrofit">Type-safe HTTP-Client für Android und Java von Square, Inc.</string>
     <string name="glide">Glide</string>
     <string name="about_glide">Eine Bibliothek zum Laden und Cachen von Bildern mit Fokus auf ruckelloses Scrollen</string>
-    <string name="circular_progress_view">KreisProgressAnsicht</string>
-    <string name="about_circular_progress_view">Kreisförmige Progresszeile im Material Design</string>
-    <string name="circle_image_view">KreisBildAnsicht</string>
+    <string name="circular_progress_view">CircularProgressView</string>
+    <string name="about_circular_progress_view">Kreisförmige Fortschrittsanzeige im Material Design</string>
+    <string name="circle_image_view">CircleImageView</string>
     <string name="about_circle_image_view">Runde Bilder für Android</string>
-    <string name="photo_view">FotoAnsicht</string>
-    <string name="about_photo_view">Ein Widget zur Vorschau und Skalierung des Bildes</string>
+    <string name="photo_view">PhotoView</string>
+    <string name="about_photo_view">Ein Widget zur Vorschau und Skalierung von Bildern</string>
     <string name="page_indicator">InkPageIndicator</string>
     <string name="about_page_indicator">InkPageIndicator erstellt von @nickbutcher für Plaid https://github.com/nickbutcher/plaid und gebackportet von David Pacioianu für API 14+ (4.0+).</string>
     <string name="greendao_db">greenDAO</string>
     <string name="about_greendao_db">greenDAO ist eine leichte und schnelle Lösung für Android, die Objekte zu SQLite Datenbanken abbildet.</string>
-
+    <string name="butter_knife">Butter Knife</string>
+    <string name="about_butter_knife">Verbinde Android views und callbacks mit Feldern und Methoden.</string>
```

Comment:
Please write views and callbacks with capital letters in German. Also [durchkoppeln](https://de.wikipedia.org/wiki/Durchkopplung) bitte…

"Android-Views"